src/components/Icon/Icon.scss

```diff
@@ -364,9 +364,13 @@
 }
 
 // New design language color styles
-
 .colorBase {
   @include color-icon('ink');
+
+  &::before {
+    background-color: var(--p-surface-neutral);
```

Comment:
This is on master. Should there be a fallback background for when people are using a new DL color don't have the newDesignLanguage feature enabled?

Reply:
The only colors I didn't add a backdrop to are base and primary.

Reply:
In that case we should probably remove them from the list of colors that support backdrops in https://github.com/Shopify/polaris-react/blob/5efa9c7447d2aea91c8612eb22151ed6be069351/src/components/Icon/Icon.tsx#L11-L27